protocol/src/main/protobuf/User.proto

```diff
@@ -439,6 +439,211 @@ message CreatePreparedStatementResp {
 }
 
 /*
+ * Request message for getting server metadata
+ */
+message GetServerMetaReq {
+}
+
+enum CollateSupport {
```

Comment:
For all enums below, use UNKNOWN as the zero-th value to avoid the problem [mentioned here](http://androiddevblog.com/protocol-buffers-pitfall-adding-enum-values/)?